core/src/main/java/lucee/runtime/db/HSQLDBHandler.java

```diff
@@ -69,6 +70,23 @@
 	Executer executer = new Executer();
 	QoQ qoq = new QoQ();
 	private static Object lock = new SerializableObject();
+	private static boolean hsqldbDisable;
+	private static boolean hsqldbDebug;
+
+	static {
+		try {
+			hsqldbDisable = Caster.toBooleanValue(SystemUtil.getSystemPropOrEnvVar("lucee.qoq.hsqldb.disable", "false"));
+		}
+		catch (PageException e) {
+			hsqldbDisable = false;
+		}
+		try {
+			hsqldbDebug = Caster.toBooleanValue(SystemUtil.getSystemPropOrEnvVar("lucee.qoq.hsqldb.debug", "false"));
+		}
+		catch (PageException e) {
+			hsqldbDebug = false;
+		}
```

Comment:
the try catch is not necessary in this case. "getSystemPropOrEnvVar" does not throw an error and 
 "Caster.toBooleanValue" can be changed as follow to avoid an exception
`hsqldbDisable = Caster.toBooleanValue(SystemUtil.getSystemPropOrEnvVar("lucee.qoq.hsqldb.disable", "false"),false);`
so set a default value for the method `toBooleanValue`

Reply:
this is a small change i will do it manually

Reply:
That makes sense.  I was not aware using the overloaded method with the default value would avoid the checked exception, but in retrospect this makes sense.